ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/ResetServiceIT.scala

```diff
@@ -57,16 +54,13 @@ class ResetServiceIT
       "return a new ledger ID" in allFixtures { ctx =>
         for {
           lid1 <- ctx.ledgerIdentityService.getLedgerIdentity(GetLedgerIdentityRequest())
-          lid1SecondInstance <- ctx.ledgerIdentityService.getLedgerIdentity(
-            GetLedgerIdentityRequest())
+          lid1Bis <- ctx.ledgerIdentityService.getLedgerIdentity(GetLedgerIdentityRequest())
```

Comment:
the `ctx.ledgerIdentityService.getLedgerIdentity` calls are not needed anymore I think. Everything should be doable just by using `reset` and getting the ledger ids from the new contexts. Please rewrite it in such style.

Reply:
Cleaned it up.